src/results/electric_storage.jl

```diff
@@ -28,18 +28,25 @@
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 # *********************************************************************************
 """
-    add_electric_storage_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
-
-Adds the Storage results to the dictionary passed back from `run_reopt` using the solved model `m` and the `REoptInputs` for node `_n`.
-Note: the node number is an empty string if evaluating a single `Site`.
-
-Storage results:
+`Storage` results keys:
 - `size_kw` Optimal inverter capacity
 - `size_kwh` Optimal storage capacity
 - `year_one_soc_series_pct` Vector of normalized (0-1) state of charge values over the first year
 - `year_one_to_load_series_kw` Vector of power used to meet load over the first year
-- `year_one_to_grid_series_kw` Vector of power exported to the grid over the first year
-- `initial_capital_cost` Upfront capital cost for storage and inverter.
+- `initial_capital_cost` Upfront capital cost for storage and inverter
+# The following results are reported if storage degradation is modeled
+- `state_of_health`
+- `maintenance_cost`
+- `replacement_month`
+# The following are reported for mpc scenarios:
```

Comment:
@NLaws how do you feel about including these mpc results keys?

Reply:
I think that it would be better to add a docs/src/mpc/outputs.md and document the MPC outputs there (using doc strings in src/mpc/results.jl)

Reply:
addressed as of https://github.com/NREL/REopt.jl/pull/82/commits/0355286c81a25a8cebe1ac07930f497b4c0f2063